libparsec/crates/types/src/local_manifest.rs

```diff
@@ -185,6 +200,36 @@ impl Chunk {
         self.block().is_some()
     }
 
+    /// A pseudo-block is a chunk which data could be upload to form a valid
+    /// block.
+    ///
+    /// For instance let's consider a file being modified by the user:
+    ///
+    /// 1. The user creates a new file with a blocksize of 512 bytes.
+    /// 2. The user writes 512 bytes of data.
+    /// 3. The user writes 200 bytes at the offset 100.
+    ///
+    /// Up until step 3, the local file manifest contains a single chunk of 400
```

Comment:
Shouldn't it be 512 here?
```suggestion
    /// Up until step 3, the local file manifest contains a single chunk of 512
```